extensions/wikia/PortableInfobox/tests/PortableInfoboxParserTagControllerTest.php

```diff
@@ -201,6 +214,127 @@ public function testGetLayoutDataProvider() {
+										<h2 class="pi-item pi-item-spacing pi-title" style="background-color:#fff;">test</h2>
+									</aside>',
+				'text' => '<title><default>test</default></title>',
+				'templateInvocation' => [],
+				'message' => 'accent-color-default set'
+			],
+			[
+				'params' => [ 'accent-color-source' => 'color-source' ],
+				'expectedOutput' => '<aside class="portable-infobox pi-background pi-theme-wikia pi-layout-default">
+										<h2 class="pi-item pi-item-spacing pi-title" style="background-color:#000;">test</h2>
+									</aside>',
+				'text' => '<title><default>test</default></title>',
+				'templateInvocation' => [
+					'color-source' => '#000'
+				],
+				'message' => 'accent-color-source set'
+			],
+			[
+				'params' => [
+					'accent-color-default' => '#fff' ,
+					'accent-color-source' => 'color-source'
+				],
+				'expectedOutput' => '<aside class="portable-infobox pi-background pi-theme-wikia pi-layout-default">
+										<h2 class="pi-item pi-item-spacing pi-title" style="background-color:#000;">test</h2>
+									</aside>',
+				'text' => '<title><default>test</default></title>',
+				'templateInvocation' => [
+					'color-source' => '#000'
+				],
+				'message' => 'accent-color-default and accent-color-source set'
+			],
+			[
+				'params' => [ 'accent-color-text-default' => '#fff' ],
+				'expectedOutput' => '<aside class="portable-infobox pi-background pi-theme-wikia pi-layout-default">
+										<h2 class="pi-item pi-item-spacing pi-title" style="color:#fff;">test</h2>
+									</aside>',
+				'text' => '<title><default>test</default></title>',
+				'templateInvocation' => [],
+				'message' => 'accent-color-text-default set'
+			],
+			[
+				'params' => [
+					'accent-color-text-default' => '#fff' ,
+					'accent-color-text-source' => 'color-source'
+				],
+				'expectedOutput' => '<aside class="portable-infobox pi-background pi-theme-wikia pi-layout-default">
+										<h2 class="pi-item pi-item-spacing pi-title" style="color:#000;">test</h2>
+									</aside>',
+				'text' => '<title><default>test</default></title>',
+				'templateInvocation' => [
+					'color-source' => '#000'
+				],
+				'message' => 'accent-color-text-source set'
+			],
+			[
+				'params' => [
+					'accent-color-text-default' => '#fff' ,
+					'accent-color-text-source' => 'color-source'
+				],
+				'expectedOutput' => '<aside class="portable-infobox pi-background pi-theme-wikia pi-layout-default">
+										<h2 class="pi-item pi-item-spacing pi-title" style="color:#000;">test</h2>
+									</aside>',
+				'text' => '<title><default>test</default></title>',
+				'templateInvocation' => [
+					'color-source' => '#000'
+				],
+				'message' => 'accent-color-text-default and accent-color-text-source set'
+			],
+			[
+				'params' => [
+					'accent-color-text-default' => '#fff' ,
+					'accent-color-text-source' => 'color-source',
+					'accent-color-default' => '#fff' ,
+					'accent-color-source' => 'color-source2'
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "color-source2" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)